### PR TITLE
Fix landing page mobile issues

### PR DESCRIPTION
**Pull request by Necromant1k**
Fixed mobile issues. 
Added hamburger menu.

### CODE REVIEW

**Comment by patroqueeet** [comment]
Dear,

thanks for your pull request. Please see detailed feedback below:

spacing issue:
https://www.dropbox.com/s/rxz4vz5voro3moo/Screenshot%202016-11-14%2022.18.14.png?dl=0

lines and hamburger are not base-aligned
https://www.dropbox.com/s/k5r4m98x2af0mew/Screenshot%202016-11-14%2022.19.10.png?dl=0

duplicate screenshots:
https://www.dropbox.com/s/iqrvq4ev8icl6ru/Screenshot%202016-11-14%2022.19.41.png?dl=0

would you consider this font-size a great fit?
https://www.dropbox.com/s/t6ildl9zd90ocaf/Screenshot%202016-11-14%2022.20.05.png?dl=0
https://www.dropbox.com/s/u8vfu1l76aeuchu/Screenshot%202016-11-14%2022.20.36.png?dl=0

words are cut/broken in the middle
https://www.dropbox.com/s/7nveqw4y9q4pok0/Screenshot%202016-11-14%2022.20.58.png?dl=0

mobile form design (button/field):
https://www.dropbox.com/s/n8tz4881j24mkfl/Screenshot%202016-11-14%2022.21.23.png?dl=0

File: site/static/compiled/stylesheets/screen.css [hunk 1]
```diff
@@ -2606,12 +2606,12 @@ html, body {
...
   width: 1000px;
 }
-/* line 97, ../../../../../../../var/lib/gems/2.3.0/gems/zen-grids-2.0.3/sass/zen-grids/_grids.scss */
+/* line 97, ../../../../../.rvm/gems/ruby-2.3.0/gems/zen-grids-2.0.3/sass/zen-grids/_grids.scss */
 .core .container:before {
   content: "";
   display: table;
 }
-/* line 101, ../../../../../../../var/lib/gems/2.3.0/gems/zen-grids-2.0.3/sass/zen-grids/_grids.scss */
+/* line 101, ../../../../../.rvm/gems/ruby-2.3.0/gems/zen-grids-2.0.3/sass/zen-grids/_grids.scss */
```
Comment (patroqueeet) [inline site/static/compiled/stylesheets/screen.css]: those comments are making the commit look bad. can we remove them?

File: site/static/sass/screen.sass [hunk 2]
```diff
@@ -279,8 +303,21 @@ div#auth
     .container
       a
         margin: 0 0.5em 0 0.5em
+      .navigation-bar-container
```
Comment (patroqueeet) [inline site/static/sass/screen.sass]: what did you do here? div#auth is not on the landing page, isn't it?

Comment (patroqueeet) [inline site/static/sass/screen.sass]: ignore it. my fault. missed the line jump in the review interface

**Comment by patroqueeet** [comment]
Dear,

here is my feedback: it generally looks very nice. Great job! Some very small issues were observed:

white button cut on the bottom:
https://www.dropbox.com/s/0kz75wl75c0ftyt/Screenshot%202016-11-15%2023.56.42.png?dl=0

feel free to make the field and the button wider:
https://www.dropbox.com/s/pkkd7y25rrusev9/Screenshot%202016-11-15%2023.57.38.png?dl=0

I tested safari, firefox, chrome. once these two things are done, we are ready to merge here and close the issue.

again, well done...

File: site/static/sass/screen.sass [hunk 3]
```diff
@@ -83,6 +83,10 @@ html, body
   p
     margin: 0 0 10px
 
+  button
```
Comment (patroqueeet) [inline site/static/sass/screen.sass]: this broke our design: https://www.dropbox.com/s/6bacjezo2ifev44/Screenshot%202016-11-16%2000.05.03.png?dl=0 

see the "Abbrechen" button? I had a border before

Comment (Necromant1k) [inline site/static/sass/screen.sass]: Got it 
I'm sorry for this

**Comment by patroqueeet** [comment]
finally all done. very nice

**Pull request merged**